### ISSUE TITLE
Chrome Version 75.0.3770.100 (Official Build) (64-bit) doesn't go to login screen and no ability to logout

### ISSUE THREAD

**Issue by StewAlexanderACC** [netdisco/netdisco#615]
Hi all

I have Chrome Version 75.0.3770.100 (Official Build) (64-bit) on Windows 10.

When I go to the NetDisco IP & port number, NetDisco comes up, but I have no way to login to a new user or logout of my current user.

Here's the version info:

![image](https://user-images.githubusercontent.com/48565067/60624650-d634c480-9db3-11e9-945d-5b6efbe7d515.png)

This is an an Ubuntu 18.04 docker host VM running on ProxMox

There is no way to logout under guest
![image](https://user-images.githubusercontent.com/48565067/60624808-404d6980-9db4-11e9-8d9f-403fe3476d46.png)

Literally typing http://10.210.88.10:5000/# and or http://10.210.88.10:5000/login brings up the same page in Chrome.

Now, when I use Internet Explorer and type http://10.210.88.10:5000/login, I do come up with a user name and password prompt as I expect

Figuring something was cached I went into the Chrome Settings and deleted everything, and tried again to get to the website, again without typing anything immediately connected as guest, with no ability to **logout** or **log in** as a different user.

Is there something I am missing, or is this a bug?

**Comment by pyro3d** [comment]
Do you have no_auth configured in your deployment.yml ?
(see https://github.com/netdisco/netdisco/wiki/Configuration#no_auth )

**Comment by inphobia** [comment]
> Do you have no_auth configured in your deployment.yml ?
> (see https://github.com/netdisco/netdisco/wiki/Configuration#no_auth )

if this is a docker setup also see https://github.com/netdisco/netdisco-docker#docker-deployment

> The web frontend is configured to allow unauthenticated access with full admin rights. We suggest you visit the Admin -> User Management menu item, and edit no_auth in deployment.yml, to remove this guest account and set up authenticated user access.

**Comment by StewAlexanderACC** [comment]
I have setup other users under Admin -> User Management, which was great, but I then had no access to the database until I tried to use Internet Explorer, logging in via the login screen, and granting guest admin privalleges

**Comment by StewAlexanderACC** [comment]
I haven't changed anything in the deployment.yml config script because the directions I found were very hard to follow

**Comment by StewAlexanderACC** [comment]
Easier in the dockerhub area, sigh... let me try this

**Comment by StewAlexanderACC** [comment]
How do I run the changes to the deployment.yml after updating it?

**Comment by inphobia** [comment]
> How do I run deployment.yml, as in get NetDisco to accept the changes to the deployment.yml file after updating it @inphobia?

netdisco should pick up those changes automatically, but dunno if that's been tested in containers. i would suggest to restart the containers after changing deployment.yml

**Comment by StewAlexanderACC** [comment]
Well it is not automatic, I'll try restarting the docker containers

**Comment by StewAlexanderACC** [comment]
@inphobia / @ollyg 

I set in the deployment.yml no_auth to false
```
database:
  name: 'netdisco'
  user: 'netdisco'
  pass: 'netdisco'
  #host: 'localhost'

# set to true to globally disable authentication/login.
# the user called "guest" has full admin/port-control rights.
# ```````````````````````````````````````````````````````````
no_auth: false
```
But Chrome still brings me directly to the guest account, already logged in

**Comment by StewAlexanderACC** [comment]
Internet Explorer takes me to the login prompt

**Comment by StewAlexanderACC** [comment]
Can one of you developers attempt to replicate the error so you can see it for yourself?

**Comment by StewAlexanderACC** [comment]
Interestingly there is now a "logout" under the guest account menu in Chrome, that wasn't there before

![image](https://user-images.githubusercontent.com/48565067/60722499-1bb1d880-9eff-11e9-85f6-10bdc0e363c1.png)

Still, it is odd that Chrome does not come to the login screen automatically

**Comment by ollyg** [comment]
Probably what happens is that your browser had a cookie from when no_auth is true, so even though you changed the setting the cookie remained valid and accepted.

If you log out I think it should start working as desired 🤞

**Comment by StewAlexanderACC** [comment]
@ollyg 

Yes, performing as expected.

* Might want make a note however of the behavior in case others see this (which I am assuming would be true)

Thank you for all of your help, please close this ticket

**Comment by inphobia** [comment]
can this be reproducded in private browsing mode in chrome? sounds like either:
 * some kind of plugin/password manager in chrome
 * form autocompletion
 * cookie reuse

now, i'm a firefox guy so i tested with that. the only cookie thats set for a normal login in the dancer.session one, which expires at the end of the session. (side note, my firefox is also configured to clear all active logins when closed).


i guess we could regenerate the dancer session cookie when the auth setting gets changed, but seems like a lot of work....

https://github.com/netdisco/netdisco-docker/blob/233032ed30928bd658d611c9db447eba6cabd902/netdisco-postgresql/netdisco-updatedb.sh#L39-L42
